src/frontend/js/widgets/Dashboard/components/DashboardItem/Order/DashboardItemOrder.tsx

```diff
@@ -299,11 +325,57 @@ const OrganizationBlock = ({ order, product }: { order: CredentialOrder; product
             </div>
           </div>
         )}
+        <PaymentItem />
       </div>
     </div>
   );
 };
 
+const PaymentItem = () => {
```

Comment:
I'll use the term `Installment` instead of `Payment` that is too generic IMO.
Or `PaymentSchedule` ?